eng/pipelines/templates/steps/use-node-test-version.yml

```diff
@@ -62,4 +62,4 @@ steps:
       }
 
     displayName: Reinstall native dependencies
-    condition: ne(variables['NodeTestVersion'], variables['NodeVersion'])
```

Comment:
We should be able to remove this condition now.  I'd rather always reinstall native deps to make sure it keeps working on all OS/node combinations, even if it's unnecessary if node version is the same.